src/main/java/core/basesyntax/model/Banana.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.model;
+
+import java.time.LocalDate;
+
+public class Banana extends Fruit {
```

Comment:
each of your fruits doesn't have anything that differs from each other except name, so creating separate class for each fruit looks redundant, you can just have Fruit with field name, or type